### ISSUE TITLE
undefined reference to symbol 'eglCreateWindowSurface'

### ISSUE THREAD

**Issue by pskora** [DavidEGrayson/ahrs-visualizer#3]
New instalation of raspbian plus dependencies installed for ahrs-visualizer. 

Ahrs-visualizer installed from:
http://www.davidegrayson.com/minimu9-ahrs/debian/ahrs-visualizer_1.0.1-1_armhf.deb
not showing main window.

From source:
pi@raspberrypi ~/ahrs-visualizer $ ls
ahrs-visualizer.1    ahrs-visualizer.d  assets       Makefile      model_board      png_texture.d  png_texture.o   version.h
ahrs-visualizer.cpp  ahrs-visualizer.o  LICENSE.txt  mock_ahrs.py  png_texture.cpp  png_texture.h  README.textile
pi@raspberrypi ~/ahrs-visualizer $ make
cc -L/opt/vc/lib -lGLESv2 -lpng12 -lboost_program_options  ahrs-visualizer.o png_texture.o model_board/model_board.o   -o ahrs-visualizer
/usr/bin/ld: ahrs-visualizer.o: undefined reference to symbol 'eglCreateWindowSurface'
//opt/vc/lib/libEGL.so: error adding symbols: DSO missing from command line
collect2: ld returned 1 exit status
<wbudowane>: polecenia dla obiektu 'ahrs-visualizer' nie powiodły się
make: *** [ahrs-visualizer] Błąd 1

pi@raspberrypi ~/ahrs-visualizer $ cat /etc/debian_version 
7.8

**Comment by DavidEGrayson** [comment]
I found a similar issue here: https://github.com/raspberrypi/firmware/issues/290

Could you go into the Makefile and add these options to the command that was giving the error (the linking step)?

    -lbcm_hosts -lEGL

**Comment by pskora** [comment]
Problem with compilation:
LDFLAGS += -L/opt/vc/lib -lGLESv2 -lbcm_host -lEGL -lm -lstdc++
I changed this line above (-lbcm_host -lEGL -lm -lstdc++  - this was added). Now the compilation is working good :)

Problem with view:
Problem with view was such that I shouldn't try to open this via xorg and network (ssh -X rasperry_pi_host). When I was running this program normally on tv connected to rpi was working good.

Tested on newest version of Raspbian (2015-05-05-raspbian-wheezy)
